base/workflow/R/run.write.configs.R

```diff
@@ -22,9 +22,14 @@
 #' @export
 #'
 #' @author David LeBauer, Shawn Serbin, Ryan Kelly, Mike Dietze
-run.write.configs <- function(settings, write = TRUE, ens.sample.method = "uniform", 
+run.write.configs <- function(settings, input_design, write = TRUE,  
                               posterior.files = rep(NA, length(settings$pfts)), 
                               overwrite = TRUE) {
+  
+  if(is.null(input_design)) {
```

Comment:
Unless you default input_design to NULL I don't think this check will ever be true. It's exceedingly unlikely that anyone will manually pass input_design = NULL into the function. As such I think this check can be dropped